<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frameWorkflow">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="radioButtonCasePlanning">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Case Planning</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroupWorkflow</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonPreOpRIOCheck">
           <property name="text">
            <string>Pre Op RIO Check</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroupWorkflow</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonFemoralPrep">
           <property name="text">
            <string>Femoral Prep</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroupWorkflow</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonAcetabularPrep">
           <property name="text">
            <string>Acetabular Prep</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroupWorkflow</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonFinalResult">
           <property name="text">
            <string>Final Results</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroupWorkflow</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonUser">
        <property name="text">
         <string>...</string>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelUser">
        <property name="text">
         <string>User</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonAdmin">
        <property name="text">
         <string>...</string>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonScreenshot">
        <property name="text">
         <string>...</string>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonShutDown">
        <property name="text">
         <string>...</string>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonOptions">
        <property name="text">
         <string>Options</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupOptionsOrScreen</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonScreen">
        <property name="text">
         <string>Screen</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupOptionsOrScreen</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidgetViewer">
      <widget class="LoginDialog" name="pageLogin"/>
      <widget class="CaseManagementDialog" name="pageCaseManagement"/>
      <widget class="FinalResultsOpenGLNativeWidget" name="pageFinalResult"/>
      <widget class="QWidget" name="pageMultiWidget">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="THAStdMultiWidget" name="multiWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frameViewAndMode">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QWidget" name="layoutWidget1">
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QToolButton" name="toolButtonZoom">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources/resources.qrc">
                   <normaloff>:/icons/toolbar/zoom.png</normaloff>:/icons/toolbar/zoom.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroupControl</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButtonLuminance">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources/resources.qrc">
                   <normaloff>:/icons/toolbar/luminance.png</normaloff>:/icons/toolbar/luminance.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroupControl</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButtonContrast">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources/resources.qrc">
                   <normaloff>:/icons/toolbar/contrast.png</normaloff>:/icons/toolbar/contrast.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroupControl</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QmitkLevelWindowWidget" name="levelWindow" native="true">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>400</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="layoutWidget2">
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QToolButton" name="toolButtonPrepOpMode">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources/resources.qrc">
                   <normaloff>:/icons/toolbar/pre-op-mode.png</normaloff>:/icons/toolbar/pre-op-mode.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroupMode</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButtonCupPlanMode">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources/resources.qrc">
                   <normaloff>:/icons/toolbar/cup-plan-mode.png</normaloff>:/icons/toolbar/cup-plan-mode.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroupMode</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButtonStemPlanMode">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources/resources.qrc">
                   <normaloff>:/icons/toolbar/stem-plan-mode.png</normaloff>:/icons/toolbar/stem-plan-mode.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroupMode</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButtonReducedMode">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources/resources.qrc">
                   <normaloff>:/icons/toolbar/reduced-mode.png</normaloff>:/icons/toolbar/reduced-mode.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroupMode</string>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="layoutWidget">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QToolButton" name="toolButtonReamingView">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources/resources.qrc">
                   <normaloff>:/icons/toolbar/reaming-view.png</normaloff>:/icons/toolbar/reaming-view.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroupView</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButtonCTView">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources/resources.qrc">
                   <normaloff>:/icons/toolbar/ct-view.png</normaloff>:/icons/toolbar/ct-view.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroupView</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton3DSlicerView">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources/resources.qrc">
                   <normaloff>:/icons/toolbar/3d-slicer-view.png</normaloff>:/icons/toolbar/3d-slicer-view.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroupView</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButtonXRayView">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources/resources.qrc">
                   <normaloff>:/icons/toolbar/x-ray-view.png</normaloff>:/icons/toolbar/x-ray-view.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroupView</string>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageImage">
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="ImageWidget" name="imageWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <widget class="QWidget" name="pageEmpty">
         <layout class="QVBoxLayout" name="verticalLayout_6"/>
        </widget>
        <widget class="CasePlanningWidget" name="pageCasePlanning"/>
        <widget class="FemoralPrepWidget" name="pageFemoralPrep"/>
        <widget class="AcetabularPrepWidget" name="pageAcetabularPrep"/>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonBack">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonNext">
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="action_Pelvis_Checkpoint">
   <property name="text">
    <string>&amp;Pelvis Checkpoint</string>
   </property>
  </action>
  <action name="action_Pelvis_Landmark">
   <property name="text">
    <string>&amp;Pelvis Landmark</string>
   </property>
   <property name="toolTip">
    <string>Pelvis_Landmark</string>
   </property>
  </action>
  <action name="action_Pelvis_Registration">
   <property name="text">
    <string>&amp;Pelvis Registration</string>
   </property>
   <property name="toolTip">
    <string>Pelvis Registration</string>
   </property>
  </action>
  <action name="action_RIO_Registratoin">
   <property name="text">
    <string>&amp;RIO Registratoin</string>
   </property>
   <property name="toolTip">
    <string>RIO Registratoin</string>
   </property>
  </action>
  <action name="action_Implant_Planning">
   <property name="text">
    <string>&amp;Implant Planning</string>
   </property>
   <property name="toolTip">
    <string>Implant Planning</string>
   </property>
  </action>
  <action name="action_Broach_Tracking">
   <property name="text">
    <string>&amp;Broach Tracking</string>
   </property>
   <property name="toolTip">
    <string>Broach Tracking</string>
   </property>
  </action>
  <action name="action_Pelvis_CT_Landmark">
   <property name="text">
    <string>&amp;Pelvis CT Landmark</string>
   </property>
   <property name="toolTip">
    <string>Pelvis CT Landmark</string>
   </property>
  </action>
  <action name="action_Cup_Reaming">
   <property name="text">
    <string>&amp;Cup Reaming</string>
   </property>
   <property name="toolTip">
    <string>Cup Reaming</string>
   </property>
  </action>
  <action name="action_Cup_Impaction">
   <property name="text">
    <string>&amp;Cup Impaction</string>
   </property>
   <property name="toolTip">
    <string>Cup Impaction</string>
   </property>
  </action>
  <action name="action_Login">
   <property name="text">
    <string>&amp;Login</string>
   </property>
  </action>
  <action name="action_Manage_Case">
   <property name="text">
    <string>&amp;Manage Case</string>
   </property>
  </action>
  <action name="action_Final_Result">
   <property name="text">
    <string>&amp;Final Result</string>
   </property>
   <property name="toolTip">
    <string>Final Result</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>THAStdMultiWidget</class>
   <extends>QWidget</extends>
   <header>THAStdMultiWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkLevelWindowWidget</class>
   <extends>QWidget</extends>
   <header>QmitkLevelWindowWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QWidget</extends>
   <header>ImageWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LoginDialog</class>
   <extends>QWidget</extends>
   <header>LoginDialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CaseManagementDialog</class>
   <extends>QWidget</extends>
   <header>CaseManagementDialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CasePlanningWidget</class>
   <extends>QWidget</extends>
   <header>CasePlanningWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AcetabularPrepWidget</class>
   <extends>QWidget</extends>
   <header>AcetabularPrepWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FinalResultsOpenGLNativeWidget</class>
   <extends>QWidget</extends>
   <header>FinalResultsOpenGLNativeWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FemoralPrepWidget</class>
   <extends>QWidget</extends>
   <header>FemoralPrepWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupWorkflow"/>
  <buttongroup name="buttonGroupOptionsOrScreen"/>
  <buttongroup name="buttonGroupControl"/>
  <buttongroup name="buttonGroupView">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="buttonGroupMode">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
